src/components/confirmation/ConfirmationPanel.tsx

```diff
@@ -0,0 +1,29 @@
+const ConfirmationPanel = () => {
+  return (
+    <div className="flex flex-col mt-40 self-center w-[321px]">
+      <img
+        loading="lazy"
+        src="src/assets/smilies.svg"
+        className="aspect-[1.71] object-contain object-center w-[133px] max-w-full self-center"
+        alt="Image 2"
+      />
+      <h2 className="text-black text-xl font-bold tracking-3 mt-4 self-center">
+        Yey!
+      </h2>
+      <p className="text-neutral-800 text-lg font-semibold leading-8 mt-4 self-center">
+        Only one more step to start enjoying
+      </p>
+      <p className="text-neutral-800 text-lg font-semibold leading-8 mt-4 self-center">
+        T A R G E T
+      </p>
```

Comment:
It could be a good idea to achieve this style with classes instead of using spaces

Reply:
Like Eugenia said, you can use the "tracking" prop of tailwind here to apply the letter spacing